requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Package
-httpx==0.16.1
+httpx>=0.18.0,<0.19.0
```

Comment:
For development and tests you probably want to pin requirements to exact version.

Reply:
Updated. 

Noticed events does not get processed (hence motion sensor does not work in homeassistant). Think it's not related to httpx version. Will look into it further